app/components/support_interface/application_choice_component.rb

```diff
@@ -33,7 +38,6 @@ def rows
         end
       end
 
-      rows << { key: 'Feedback', value: application_choice.rejection_reason } if application_choice.rejection_reason.present?
```

Comment:
Assume we are removing this as it's duplicated.

Reply:
Yes — the other field falls back correctly: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3877/commits/d61fd89050ba4114e3eb1e2cd8c74db4ccbf5c3d